examples/plot_scalable_poly_kernels.py

```diff
@@ -0,0 +1,119 @@
+"""
+=======================================================
+Scalable learning with polynomial kernel aproximation
+=======================================================
+
+This example illustrates the use of :class:`PolynomialSampler` to efficiently
+generate polynomial kernel feature-space approximations, which can be used
+to train linear classifiers that approximate the accuracy of kernelized ones.
+
+.. currentmodule:: sklearn.kernel_approximation
+
+We use the Covtype dataset [2], trying to reproduce the experiments on the
+original paper of Tensor Sketch [1] (i.e., the algorithm implemented by
+:class:`PolynomialSampler`).
+
+First, we compute the accuracy of a linear classifier on the original
+features. Then, we train linear classifiers on different numbers of features
+generated by :class:`PolynomialSampler`, approximating the accuracy of a
+kernelized classifier in a scalable manner.
+"""
+
+
+##############################################################################
+#
+# ############################################################################
+#
+# First of all, load some data manipulation functions:
+
+# Author: Daniel Lopez-Sanchez <lope@usal.es>
+# License: BSD 3 clause
+from sklearn.datasets import fetch_covtype
+from sklearn.model_selection import train_test_split
+from sklearn.preprocessing import normalize, MinMaxScaler
+
+##############################################################################
+# Then load the linear classifier that we will use, and the class for
+# polynomial kernel feature-map approximation:
+
+from sklearn.svm import LinearSVC
+from sklearn.kernel_approximation import PolynomialSampler
+
+##############################################################################
+# Load the Covtype dataset and transform into a binary classification problem
+# to match the format of the dataset in the LIBSVM webpage [2].
+
+data = fetch_covtype()
+X, Y = data["data"], data["target"]
+
+Y[Y != 2] = 0
+Y[Y == 2] = 1
```

Comment:
- [ ] Described data a bit more.

The reason for using `Y[Y == 2] = 1`, is because we try to reproduce one of the results from the original Tensor Sketch paper [1], where they used the [binary version of the Covertype dataset](https://www.csie.ntu.edu.tw/~cjlin/libsvmtools/datasets/binary.html), as opposed to the standard 6-classes version. From the paper were this dataset was created [2]:

> We modified the 7-class classification problem into a binary classification problem where the goal was to separate class 2 from the other 6 classes.

So yeah, a bit of paper archeology. :man_scientist: 

[1] Pham, N., & Pagh, R. (2013, August). Fast and scalable polynomial kernels via explicit feature maps. In Proceedings of the 19th ACM SIGKDD international conference on Knowledge discovery and data mining (pp. 239-247)

[2] R. Collobert, S. Bengio, and Y. Bengio. A parallel mixture of SVMs for very large scale problems. Neural Computation, 14(05):1105-1114, 2002.